spec/classes/core/openvpn_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'profile::core::openvpn' do
+  on_supported_os.each do |os, os_facts|
+    next unless os =~ %r{almalinux-9-x86_64}
```

Comment:
There should be spec coverage of the letsencrypt resources.

Ideally, the profile tests would be does as a `shared_example`, that can also be included in the role spec file.